ansible/roles/baselayout/vars/main.yml

```diff
@@ -43,7 +43,7 @@ packages: {
   # partials/repo/centos7.yml for arm64
   centos7_arm64: ['git'], # git2u not available for aarch64 (yet)
   centos7_x64: ['devtoolset-6-libatomic-devel,git2u,centos-release-scl'],
-  centos7_ppc64: ['devtoolset-6-libatomic-devel,git,python3'],
+  centos7_ppc64: ['cmake3,devtoolset-6-libatomic-devel,git,python3'],
```

Comment:
@rvagg what is the syntax of the packages section? None of these packages are currently being installed, though I'd swear they used to be. I'm not having any luck figuring out what the OS+ARCH combination string should be in the packages table, or whether these are cumulative or not. I tried `centos7_ppc64le`.

Reply:
When I logged into `centos7_ppc64le-2` earlier it had `cmake3` on it

Reply:
yes, but it doesn't get installed now, nor does git. I'm mystified.

Reply:
https://github.com/nodejs/build/issues/2321#issuecomment-624130373